ui/pages/add-nft/add-nft.js

```diff
@@ -153,9 +158,12 @@ export default function AddNft() {
                     >
                       {t('nftAddFailedMessage')}
                     </Typography>
-                    <button
-                      className="fas fa-times add-nft__close"
-                      title={t('close')}
+                    <ButtonIcon
+                      className="add-nft__close"
+                      iconName={ICON_NAMES.CLOSE}
+                      size={ICON_SIZES.SM}
+                      ariaLabel={t('close')}
+                      data-testid="add-nft-error-close"
                       onClick={() => setNftAddFailed(false)}
                     />
                   </Box>
```

Comment:
Would be good to get a screenshot here

Reply:
<img width="560" alt="ImportError" src="https://user-images.githubusercontent.com/46655/220722048-6e3f4105-1c49-4771-be75-c7d408213f79.png">